environment.yml

```diff
@@ -24,4 +24,6 @@ dependencies:
 - pip:
     - git+https://github.com/HazyResearch/numbskull@40ac1af20538c17b4726c963c69adcb81314efa5
     - neo4j-driver==1.1.0
+    - psycopg2==2.7.1
```

Comment:
Get from anaconda https://anaconda.org/anaconda/psycopg2.

We always want to get a precompiled binary when possible from either the `anaconda` or `conda-forge` channel. This will make installation faster.